llvm/lib/Transforms/Utils/LoopUtils.cpp

```diff
@@ -759,6 +759,13 @@ void llvm::breakLoopBackedge(Loop *L, DominatorTree &DT, ScalarEvolution &SE,
         DomTreeUpdater DTU(&DT, DomTreeUpdater::UpdateStrategy::Eager);
         Header->removePredecessor(Latch, true);
 
+        for (PHINode &PN : Header->phis()) {
```

Comment:
Style wise, can this be done in the caller?  This routine has a clearly stated purpose, and I generally dislike routines which a bunch of vaguely related things.  The need to preserve LCSSA below is somewhat of a counter point.  

(Minor, please don't let this be blocking.)